src/custom/utils/misc.ts

```diff
@@ -118,3 +121,39 @@ export function hashCode(text: string): number {
 
   return hash
 }
+
+/**
+ * Convenient method to get the error message from the error raised by a provider.
+ *
+ * Some providers return some description in the error.message, and some others the error message is itself a String
+ * with the error message
+ */
+export function getProviderErrorMessage(error: any) {
+  return typeof error === 'string' ? error : error.message
```

Comment:
No type `error`?
Also, what if `error` doesn't have a `message`?

Reply:
`No type error?`
what do u mean? Theres no type error. I'm just checking if they provided a `string` the I check if it has a `message` property


> Also, what if error doesn't have a message?

then, it returns `undefined`

Reply:
I mean, a type for `error` param instead of `any`

Reply:
It's any on porpuse. It's the case, it can be undefined, an object, a string. Basically any type. This is also how is modeled in our libraries and our code